client/web/compose/src/components/ModuleFields/Configurator/Select.vue

```diff
@@ -140,6 +140,16 @@ export default {
     isNew () {
       return this.module.moduleID === NoID || this.field.fieldID === NoID
     },
+
+    defaultFieldOptions () {
```

Comment:
Since these are not really default field options but just the default state, can you please rename to defaultFieldOptionsState